components/thumbnail/basic/.gitignore

```diff
@@ -0,0 +1,3 @@
+
+lib
+node_modules
```

Comment:
No need as it's already in root https://github.com/SUI-Components/sui-components/blob/feature/thumbnail-basic/.gitignore

Also, i would remove this file and add lib to root's .gitignore.